SparkPost/support-docs
# EA-9269 - Add support docs for inbox-and-design-tracker pages

### What Changed

- Added pages for `inbox-and-design-tracker` pages and images.

### How To Test or Verify

- Verify `/analyst/inbox-and-design-tracker` pages exist and function (with images).
- Check pages match (except the intercom excerpts) https://support.emailanalyst.com/en/collections/1398281-inbox-design-tracker
- Check there are no refs to `intercom`, `support.emailanalyst.com`, or intercom cdn

### PR Checklist

#### All PRs Checklist


#### Content Changes Checklist
